bsoid_app/clustering.py

```diff
@@ -58,7 +58,7 @@ def show_classes(self):
         plt1.suptitle('HDBSCAN assignment')
         col1, col2 = st.beta_columns([2, 2])
         col1.pyplot(fig1)
-        fig1.savefig('hdbscan_assignment.png', dpi=300, bbox_inches='tight')
+        fig1.savefig(os.path.join(self.working_dir, 'hdbscan_assignment.png'), dpi=300, bbox_inches='tight')
```

Comment:
does it work for you to have this hardcoded or will it be annoying/have files overwritten if you run this multiple times?